spec/presenters/hyku/manifest_enabled_work_show_presenter_spec.rb

```diff
@@ -37,7 +39,30 @@
 
     it "returns a hash containing the rendering information" do
       work.rendering_ids = [work.file_sets.first.id]
-      expect(subject).to include(:sequence_rendering)
+      expect(subject).to be_an Array
+    end
+  end
+
+  describe "#manifest metadata" do
+    subject do
+      presenter.manifest_metadata
+    end
+
+    context do
+      it "returns a hash containing the metadata with a single value" do
+        expect(subject).to be_a Array
+        expect(subject[0]['value']).to eq 'Test title'
+      end
+    end
+
+    context do
+      before do
+        work.title = ['Test title', 'Another test title']
+      end
+
+      it "returns an array of metadata values" do
+        expect(subject[0]['value']).to include('Test title', 'Another test title')
+      end
```

Comment:
It would be good to add a test for the metadata pair's label as well.

Reply:
done!